### PR TITLE
Add intro to italian resources

### PR DESCRIPTION
**Pull request by marcocarraro**

### CODE REVIEW

**Review by lauportell — APPROVED** [review]

**Comment by lauportell** [comment]
Hi Marco,

thanks for adding the info for Italy!

As a suggestion, don't you think it would be better to add the Open-science.it resource in the resource table for the page instead of in the text?

However, if you think it is better in the text, I'll happily merge :)

File: pages/national_resources/it_resources.md [hunk 1]
```diff
@@ -42,12 +24,31 @@ resources:
...
+* National working groups
+
+Data management support and training for life science researchers is provided at national level by the Italian ELIXIR Node, hosted at the Italian Infrastructure for Bioinformatics ([IIB](https://elixir-italy.org/)).
+
+The Research Data Alliance ([RDA](https://www.rd-alliance.org/groups/rda-italy)) Italian Node is hosted at the CNR Institute of Information Science and Technologies (CNR-ISTI) and provides support and training for both humanities and STEM.
+
+Italian [OpenAIRE NOAD](https://www.openaire.eu/contact-noads) (National Open Access Desk) is in place in order to develop capacity at local level and to provide expert advice on infrastructures, supporting workflows for open science.
+
+### Open-science.it
+[Open-science.it](hhttps://open-science.it/home)  is the Italian portal dedicated to the field of open science (in Italian).
```
[the remark this answers is not in the document]
Comment (marcocarraro) [reply pages/national_resources/it_resources.md]: Thanks, typo fixed!

**Comment by bedroesb** [comment]
@marcocarraro  would it be possible to complete this part of the metadata section for Open-science.it?

The related pages can be tagged with relevant page_ids, which can be taken from this page: https://rdmkit.elixir-europe.org/website_overview

Please use other country pages like the one from Belgium as inspiration:
https://github.com/elixir-europe/rdmkit/blob/master/pages/national_resources/be_resources.md?plain=1


```yml
national_resources:
  - name: Open-science.it
    description:
    how_to_access: <!--- REPLACE THIS free text to explain if credentials, login, specific affiliations etc are needed to access the resource or tool--->
    instance_of: <!--- REPLACE THIS with the tool name of which this resource is an instance of, taken from the all tools and resources page --->
    related_pages:
      tool_assembly: [<!---REPLACE THIS with the page ID of the tool_assembly pages that you want to list here as related pages--->]
      your_domain: [<!---REPLACE THIS with the page ID of the domain pages that you want to list here as related pages--->]
      your_role: [<!---REPLACE THIS with the page ID of the your_role pages that you want to list here as related pages--->]
      your_tasks: [<!---REPLACE THIS with the page ID of the your_tasks pages that you want to list here as related pages--->]
    url: https://open-science.it/
```

**Comment by marcocarraro** [comment]
> @marcocarraro would it be possible to complete this part of the metadata section for Open-science.it?
The metadata for open-science.it was added

**Comment by lauportell** [comment]
Hi @marcocarraro,

Thank you for your contribution!
The national page for Italy is already available on the RDMkit:
https://rdmkit.elixir-europe.org/it_resources

Best,
Laura.

**Pull request merged**